src/main/java/com/madadata/eval/prestoconnectivity/resource/MyResource.java

```diff
@@ -66,4 +121,83 @@ public Object tableMeta() throws Exception {
                 "table_catalog", "table_schema", "table_name", "table_type");
     }
 
+    /*
+    doesn't work
+     */
+    @GET
+    @Path("/create")
+    @Produces(MediaType.APPLICATION_JSON)
+    public int create(@QueryParam("name") @NotEmpty String name) throws Exception {
+        return jdbiCreate(String.format("create table %s (id int primary key, name varchar(100))", name));
+    }
+
+    /*
+    doesn't work
+     */
+    @GET
+    @Path("/createquery")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Object createQuery() throws Exception {
+        return createQuery("select * from hive.information_schema.tables",
+                "table_catalog", "table_schema", "table_name", "table_type");
+    }
+
+    @GET
+    @Path("/create-table")
+    @Produces(MediaType.APPLICATION_JSON)
+    public int createTable(@QueryParam("name") @NotEmpty String name) throws Exception {
+        return executeSQL(String.format("create table %s (id integer, name varchar(100))", name));
+    }
+
+    @GET
+    @Path("/insert")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Object insert() throws Exception {
+        return executeSQL("insert into madatest values (3, 'quheng_many')");
+    }
+
+    /*
+    doesn't work
+    I do not found any support about UPDATE in  https://prestodb.io/docs/current/
+    */
+    @GET
+    @Path("/update")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Object update() throws Exception {
+        return executeSQL("update madatest set id = 4 where name = quheng_many" );
+    }
+
+    @GET
+    @Path("/groupby")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Object groupby() throws Exception {
+        return query("select sum(id) as sum , name from madatest group by name", "name", "sum");
+    }
+
+    @GET
+    @Path("/orderby")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Object orderby() throws Exception {
+        return query("select id, name from madatest order by id desc", "id", "name");
+    }
+
+    /*
+    doesn't work
```

Comment:
a minor thing about JavaDoc, you either don't use them, or use proper format like:

```
/**
 * line 1
 * line 2
 */
```